tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-indexed-access.bal

```diff
@@ -88,6 +88,362 @@ function testLengthOfXMLSequence() returns [int, int, int, int] {
     return [x1.length(), x2.length(), x2[2].length(), x3.length()];
 }
 
+const i = -1;
+
+function testInvalidXMLAccessWithNegativeIndex() returns error? {
+    int a = -1;
```

Comment:
Can we add some xml sub type sequence tests as well. eg: `xml<xml:Comment>`

Reply:
I believe there is https://github.com/ballerina-platform/ballerina-lang/pull/40114/files/5ff4ac52aa7ec017f6582c9daaf91208f3566c3c#diff-d48872f73bc16a73f08f8f39ee2ac3554cfff305bb4fa861a222befe454e4147R209

Reply:
yes.